src/fondant/explore.py

```diff
@@ -148,7 +148,10 @@ def run_explorer_app(  # type: ignore  # noqa: PLR0913
         "--detach",
     ]
 
-    subprocess.call(cmd, stdout=subprocess.PIPE)  # nosec
+    try:
+        subprocess.check_call(cmd, stdout=subprocess.PIPE)  # nosec
+    except subprocess.CalledProcessError as e:
+        raise OSError(e.returncode)
```

Comment:
Why did you choose an `OSError` here? I would probably have gone for a `SystemExit` exception myself.

Reply:
Might make more sense here indeed, updated